redaxo/src/addons/phpmailer/pages/config.php

```diff
@@ -321,6 +326,12 @@
 $n['note'] = rex_i18n::rawMsg('phpmailer_archive_info', rex_mailer::logFolder(), '...'.substr(rex_mailer::logFolder(), -30));
 $formElements[] = $n;
 
+if (is_dir(rex_mailer::logFolder())) {
+    $n = [];
+    $n['field'] = '<button data-confirm="' . $addon->i18n('archive_delete_confirm') . '" class="btn btn-danger pull-right" type="submit" name="btn_delete_archive" value="' . $addon->i18n('archive_delete') . '">' . $addon->i18n('archive_delete') . '</button>';
```

Comment:
müssen wir hier ggf. escapen in den html attr.?

Reply:
welches Attribut meinst du?

Reply:
Hier in der datei gibt es einige vergleichbare stellen die auch nicht escaped werden, daher würd ich es hier auch so lassen